Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -561,8 +562,8 @@ Parser::ParseErrorOr<Selector::SimpleSelector> Parser::parse_pseudo_simple_selec
                 return ParseError::SyntaxError;
             }
             // FIXME: Support multiple, comma-separated, language ranges.
-            Vector<DeprecatedFlyString> languages;
-            languages.append(pseudo_function.values().first().token().to_string().release_value_but_fixme_should_propagate_errors().to_deprecated_string());
+            Vector<FlyString> languages;
+            languages.append(FlyString { pseudo_function.values().first().token().to_string().release_value_but_fixme_should_propagate_errors() });
```

Comment:
:thinking: wonder if we should just make the `FlyString(String)` constructor implicit, surrounding `FlyString { ... }` everywhere feels a bit noisy

Reply:
Yeah, it's something I've definitely noticed. Seems safe to cast them like this. And `DeprecatedFlyString(DeprecatedString)` is implicit and that seems to have been fine.

I'll add that as a commit.